src/wdio/services/TerraCommands/axe-command.js

```diff
@@ -30,7 +30,18 @@ const injectAxe = (axeOptions) => {
 *
 * @returns {Object} axeResults - the axe results as list of passes, violations, incomplete and inapplicable.
 */
-const runAxeTest = (rules) => {
+const runAxeTest = (axeRules) => {
+  const globalAxeRules = (browser.options.axe && browser.options.axe.options) ? browser.options.axe.options.rules : undefined;
+  // Converting `globalAxeRules` from array of rules object to object of rules.
+  const formattedGlobalRules = globalAxeRules && globalAxeRules.reduce((formattedRules, rule) => {
+    const { id, ...rest } = rule;
+    const rulesObject = formattedRules;
```

Comment:
What is happening here? You were looking to clone it?

Reply:
Yeah, I did not want to mutate the parameter, so I did want to clone it
I see your concern and have addressed here : https://github.com/cerner/terra-toolkit/pull/356/commits/50ac3e447dfe3961f6e15fdd57bf106335fd8844